Lumos/Lumos/Sources/LMMethod.swift

```diff
@@ -88,7 +88,7 @@ extension LMMethod {
         hook(option: .positionInstead, errorMessage: LMMethod.errorMessage(.cannotReplace), block: block)
     }
     
-    private func hook(option: AspectOptions, errorMessage: String, block: @escaping () -> Void) {
+    public func hook(option: AspectOptions, errorMessage: String, block: @escaping () -> Void) {
```

Comment:
Is this change intentional? Look unrelated to the PR title. But that aside, what's the rationale for making this function public?